cpu/atmega32u4/include/cpu_conf.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2017 Thomas Perrot <thomas.perrot@tupi.fr>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup         cpu_atmega32u4
+ * @{
+ *
+ * @file
+ * @brief           Implementation specific CPU configuration options
+ *
+ * @author          Thomas Perrot <thomas.perrot@tupi.fr>
+ *
+ */
+
+#ifndef CPU_CONF_H
+#define CPU_CONF_H
+
+#include "atmega_regs_common.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name Kernel configuration
+ *
+ * Since printf seems to get memory allocated by the linker/avr-libc the stack
+ * size tested sucessfully even with pretty small stacks.k
+ * @{
+ */
+#define THREAD_EXTRA_STACKSIZE_PRINTF    (128)
```

Comment:
Use (128**U**), here a below

Reply:
If I do that, I get compile errors:
/home/tperrot/dev/tprrt/RIOT/core/include/debug.h:55:79: error: comparison between signed and unsigned integer expressions [-Werror=sign-compare]
         if ((sched_active_thread == NULL) || (sched_active_thread->stack_size >= THREAD_EXTRA_STACKSIZE_PRINTF)) { \